.github/workflows/coverage-report.yml

```
@@ -30,44 +30,51 @@ jobs:
       name: github-pages
       url: ${{ steps.deployment.outputs.page_url }}
     runs-on: ubuntu-latest
-    if: ${{ !github.event.pull_request.draft }}
+    if: !github.event.pull_request.draft && github.base_ref == 'refs/heads/main'
```

actionlint

could not parse as YAML: yaml: line 33: did not find expected alphabetic or numeric character

(syntax-check)